Lib/asyncio/tasks.py

```diff
@@ -36,7 +36,10 @@ def current_task(loop=None):
     """Return a currently executed task."""
     if loop is None:
         loop = events.get_running_loop()
-    return _current_tasks.get(loop)
+    try:
+        return _current_tasks[loop]
+    except:
```

Comment:
It _shouldn't_ matter, but I think preferably we would not blanket silence all exceptions here.

```suggestion
    except KeyError:
```